### ISSUE TITLE
VersionMismatch at /combine/system  Redis transport requires redis-py versions 3.2.0 or later. You have 2.10.6

### ISSUE THREAD

**Issue by bibliotechy** [WSULib/combine-playbook#28]
Just built `v0.7` from scratch and while the combine application comes up fine, when I navigate to "combine/system", to check out the Spark and Background sessions, I get this error

```
VersionMismatch at /combine/system  
Redis transport requires redis-py versions 3.2.0 or later. You have 2.10.6
```

I noticed that the latest release of combine includes a [commit  pin the redis version](https://github.com/WSULib/combine/commit/35db8d02c43f777fa561e69db26c12b997de2eee#diff-905862eebc440ffbc8a4faab442e1b6dR115), but I wonder if something else corresponding needs to be pinned in the playbook?  

Full traceback is:
```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8081/combine/system

Django Version: 1.11.20
Python Version: 3.5.5
Installed Applications:
['core',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django_extensions',
 'django.contrib.humanize']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/transport/virtual/base.py" in create_channel
  921.             return self._avail_channels.pop()

During handling of the above exception (pop from empty list), another exception occurred:

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/django/contrib/auth/decorators.py" in _wrapped_view
  23.                 return view_func(request, *args, **kwargs)

File "/opt/combine/core/views.py" in system
  292. 	active_tasks = celery_app.control.inspect().active()

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/celery/app/control.py" in active
  119.         return self._request('active')

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/celery/app/control.py" in _request
  106.             pattern=self.pattern, matcher=self.matcher,

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/celery/app/control.py" in broadcast
  477.                     limit, callback, channel=channel,

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/pidbox.py" in _broadcast
  328.         chan = channel or self.connection.default_channel

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/connection.py" in default_channel
  852.         self.ensure_connection(**conn_opts)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/connection.py" in ensure_connection
  422.                             callback, timeout=timeout)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/utils/functional.py" in retry_over_time
  341.             return fun(*args, **kwargs)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/connection.py" in connect
  275.         return self.connection

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/connection.py" in connection
  823.                 self._connection = self._establish_connection()

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/connection.py" in _establish_connection
  778.         conn = self.transport.establish_connection()

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/transport/virtual/base.py" in establish_connection
  941.         self._avail_channels.append(self.create_channel(self))

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/transport/virtual/base.py" in create_channel
  923.             channel = self.Channel(connection)

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/transport/redis.py" in __init__
  496.         self.Client = self._get_client()

File "/usr/local/Miniconda3-4.3.30-Linux-x86_64/envs/combine/lib/python3.5/site-packages/kombu/transport/redis.py" in _get_client
  957.                 'You have {0.__version__}'.format(redis))

Exception Type: VersionMismatch at /combine/system
Exception Value: Redis transport requires redis-py versions 3.2.0 or later. You have 2.10.6
```

**Comment by ghukill** [comment]
Thanks for noting this @bibliotechy.  The pinning of the python/redis binding to `2.10.6` [stems from this issue](https://github.com/celery/celery/issues/5175), but that was cropping up on a Dockerized deployment of Combine, and the basis for this change.  Beginning to wonder now if the [Redis version installed in the Docker deploy](https://github.com/WSULib/combine-docker/blob/master/docker-compose.yml#L52) is different than Ubuntu apt-get install, causing this.

Looking into it now, and thanks again for the heads up.

**Comment by ghukill** [comment]
Fixed in v0.7.1 release.
